app/scripts/modules/core/src/managed/DeployingIntoManagedClusterWarning.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { IServerGroupCommand } from 'core/serverGroup';
+import { FormikProps } from 'formik';
+import { Application } from 'core/application';
+import { toggleResourcePause } from './toggleResourceManagement';
+
+export interface IDeployingIntoManagedClusterWarningProps {
+  app: Application;
+  formik: FormikProps<IServerGroupCommand>;
+}
+
+export const DeployingIntoManagedClusterWarning = ({ app, formik }: IDeployingIntoManagedClusterWarningProps) => {
+  const command = formik.values;
+  if (!command.resourceSummary) {
+    return null;
+  }
+  const pauseResource = React.useCallback(() => {
+    const { viewState, backingData } = formik.values;
+    toggleResourcePause(viewState.resourceSummary, app).then(() => {
+      backingData.managedResources = app.getDataSource('managedResources')?.data?.resources;
+      formik.setFieldValue('resourceSummary', null);
+    });
+  }, [app, formik]);
```

Comment:
`formik` is a stable reference, right? (i.e. doesn't get re-generated every render)